dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -397,7 +399,13 @@ MULTI_USER_REPLACEMENT_STRING="          - name: \"CHE_WORKSPACE_LOGS\"
           - name: \"CHE_KEYCLOAK_CLIENT__ID\"
             value: \"${CHE_KEYCLOAK_CLIENT__ID}\"
           - name: \"CHE_HOST\"
-            value: \"${CHE_HOST}\""
+            value: \"${CHE_HOST}\"
+          - name: \"CHE_MULTIUSER\"
+            value: \"${CHE_MULTIUSER}\"
```

Comment:
Can you elaborate on how this value is used?

Reply:
that would be needed for https://github.com/eclipse/che/pull/7188
depending on CHE_MULTIUSER true || false tomcat will be launched in appropriate mode.

Reply:
ok, thanks for the explanation